src/backend/utilities/initKilt.ts

```diff
@@ -5,6 +5,13 @@ import { logger } from './logger';
 import { trackConnectionState } from './trackConnectionState';
 
 export async function initKilt(): Promise<void> {
+  // shouldn't this check first if it is already connected?
```

Comment:
There are 3 files that use this function. Each time a new `api` is set and the connections stay open. 

While doing the **jest tests** it informs me about several process not completed. Because `api.disconnect` is never called. 

Thats why i did not use `initKilt` on my files, I did not checked if already connected and just got the api through `ConfigService.get('api')`.

Reply:
Okay, this is a good catch, I thought I have fixed that during the SDK migration which changed the behavior. [Let’s memoize it then](https://github.com/BTE-Trusted-Entity/socialkyc.io/pull/1592).

Reply:
memoize initKilt (https://github.com/BTE-Trusted-Entity/socialkyc.io/pull/1592)  solves this